web/templates/move_topic_to_stream.hbs

```diff
@@ -1,26 +1,50 @@
 {{#unless (or from_message_actions_popover only_topic_edit)}}
-<p class="white-space-preserve-wrap">{{#tr}}Move all messages in <strong>{topic_name}</strong>{{/tr}} to:</p>
+<p>
+    {{#tr}}
+    You are moving all messages from <z-stream></z-stream>
+    {{#*inline "z-stream"}}<b>{{> inline_decorated_stream_name stream=stream show_colored_icon=true}}</b>{{/inline}}
```

Comment:
This change seems to remove the `white-space-preserve-wrap`; is that intended? It seems likely that class was there for a reason.

Reply:
Yes, removing the class `white-space-preserve-wrap` was intentional. This class caused the text to preserve white spaces and wrap. After modifying the intro-text, preserving white spaces resulted in unnecessary spacing. Please refer to the attached screenshots. Thank you.

<details>
<summary>Screenshots</summary>

| Screenshot with the class applied | Screenshot with the class removed |
|-----------------------------------|-----------------------------------|
| ![Screenshot 2024-11-19 091631](https://github.com/user-attachments/assets/159827d2-0511-4590-b269-45d936a59088) | ![Screenshot 2024-11-19 091819](https://github.com/user-attachments/assets/878e3fce-8417-4f12-b05a-9c4daf187f04) |


</details>